crypto/cms/cms_kari.c

```diff
@@ -429,6 +435,7 @@ static int cms_wrap_init(CMS_KeyAgreeRecipientInfo *kari,
         return 0;
     keylen = EVP_CIPHER_key_length(cipher);
     if ((EVP_CIPHER_flags(cipher) & EVP_CIPH_FLAG_GET_WRAP_CIPHER) != 0) {
+        /* TODO: make this not get a method we can call directly */
         ret = EVP_CIPHER_meth_get_ctrl(cipher)(NULL, EVP_CTRL_GET_WRAP_CIPHER,
                                                0, &kekcipher);
```

Comment:
EVP_CTRL_GET_WRAP_CIPHER is new code in master, it seems...

Reply:
More bizarre is that *we* don't appear to use it anywhere.

Reply:
@beldmit, it looks like you added this but I can't find a reference to it either in OpenSSL or the GOST engine??

Reply:
Well... It seems to be a remnant of unsuccessful trying to add support KeyAgreement on creating the GOST CMS. As no way to pass the choice between KA and KT was found, I tried to clean up all the remnants but now I see that some still persist.

Reply:
This was a side note, BTW.  The TODO comment is sufficient for now, we can revisit this in another PR